web/libs/editor/src/components/BottomBar/Controls.tsx

```diff
@@ -98,29 +104,65 @@ export const Controls = controlsInjector<{ annotation: MSTAnnotation }>(
       ],
     );
 
-    // custom buttons replace all the internal buttons, but they can be reused if `name` is one of the internal buttons
-    if (store.customButtons?.length) {
-      for (const customButton of store.customButtons ?? []) {
+    const buttonsBefore = customButtons.get("_before");
+    const buttonsReplacement = customButtons.get("_replace");
```

Comment:
```
    const buttonsBefore = customButtons.get("_before");
    const buttonsReplacement = customButtons.get("_replace");
```
Why the difference in names, these ones have `_` but all others are just normal keys in the map. Is this supposed to be a private API?

Reply:
This way I'm trying to separate internal buttons (like `accept` and `reject`) and special places (`_before`, `_after`, `_replace`). I'll share a document soon.